client/src/pages/Signup.js

```diff
@@ -214,8 +258,26 @@ export default function SignUp() {
             </Grid>
           </form>
         </div>
+        <div className={classes.snackbar}>
+          <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
+            <Alert onClose={handleClose} severity="error">
+              {reason}
+            </Alert>
+          </Snackbar>
+        </div>
       </Grid>
       <Grid item xs={false} sm={1} md={7} className={classes.image} />
     </Grid>
   );
 }
+
+const loginUser = async (user) => {
```

Comment:
since this is essentially the same function as `Login.js`, store this function in a utils method that can be reused